src/modules/previewpane/common/cominterop/RECT.cs

```diff
@@ -11,7 +11,7 @@ namespace Common.ComInterlop
     /// The RECT structure defines a rectangle by the coordinates of its upper-left and lower-right corners.
     /// </summary>
     [StructLayout(LayoutKind.Sequential)]
-    internal struct RECT
+    public struct RECT

```

Comment:
any specific reason for this change?

Reply:
`PreviewHandler` and `FormHandlerControl` exposing `Public` methods which are using these field as parameters.